addons/calendar/static/src/js/calendar_provider_sync_wizard_form_renderer.js

```diff
@@ -0,0 +1,79 @@
+/** @odoo-module **/
+
+import { FormRenderer } from "@web/views/form/form_renderer";
+import { session } from "@web/session";
+import { useService } from "@web/core/utils/hooks";
+
+const EXTERNAL_CALENDAR_PARAMETERS = {
+    google: {
+        caption: 'Google',
+        restart_sync_method: 'restart_google_synchronization',
```

Comment:
Not ideal as this method is defined in the google_calendar module.

We could imagine having a "_prepareExternalCalendarParameters" method that receives the calendar type and is extended in sub-modules to return their own properties?

Reply:
As both use a common structure, I relied on that to require no extra code. Is that OK?